Makefile

```diff
@@ -0,0 +1,36 @@
+# Basic go commands
+GOCMD=go
+GOBUILD=$(GOCMD) build
+GOCLEAN=$(GOCMD) clean
+GOTEST=$(GOCMD) test
+GOGET=$(GOCMD) get
+GORUN=$(GOCMD) run
+
+# App info
+BINARY_NAME := baton
+VERSION := 0.1.5
+
+# Folders
+OUTPUT_FOLDER := build
+BINARY_OUTPUT := $(OUTPUT_FOLDER)/$(BINARY_NAME)
+
+# Output platforms
+PLATFORMS := windows linux darwin
+os = $(word 1, $@)
+
+# Make for all the platforms
+.PHONY: $(PLATFORMS)
+ifeq ($(OS),Windows_NT)
+$(PLATFORMS):
+	set GOOS=$(os)&& set GOARCH=amd64&& $(GOBUILD) -o $(BINARY_OUTPUT)-$(VERSION)-$(os)-amd64$(if $(filter $(os),windows),.exe,) main.go
+else
+$(PLATFORMS):
+	GOOS=$(os) GOARCH=amd64&& $(GOBUILD)-o $(BINARY_OUTPUT)-$(VERSION)-$(os)-amd64$(if $(filter $(os),windows),.exe,) naub.go

```

Comment:
I think this should follow the convention of the above command, as of now it's missing an initial `set`, had to add a space here `$(GOBUILD)-o` before the `-o` and change `naub.go` to `main.go` in order to get the build to work.

Would be nice to have some build instructions in the README 😄

Reply:
Oops, i'm sorry that must've been a typo i hadn't noticed because I use a windows machine myself